cmd_list.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"runtime"
+	"time"
+
+	proto "github.com/mattn/ft/proto"
+	"github.com/urfave/cli"
+	"google.golang.org/grpc"
+)
+
+func listFiles(ctx context.Context, client proto.FileTransferServiceClient) error {
+	slist, err := client.ListFiles(ctx, new(proto.ListRequestType))
+	if err != nil {
+		return err
+	}
+	fmt.Println("name,size,mode,modtime")
+	for {
+		file, err := slist.Recv()
+		if err != nil {
+			if err == io.EOF {
+				break
+			}
+			break
+		}
+		if runtime.GOOS != "windows" {
+			fmt.Printf("%q,\"%v\",\"%v\",\"%v\"\n",
+				file.Name, file.Size, os.FileMode(file.Mode), time.Unix(file.ModTime.Seconds, 0).Format(time.RFC3339))
+		} else {
+			fmt.Printf("%q,\"%v\",\"%o\",\"%v\"\n",
+				file.Name, file.Size, file.Mode, time.Unix(file.ModTime.Seconds, 0).Format(time.RFC3339))
+		}
```

Comment:
Why this part is separeted for Windows?

Reply:
I misunderstood that os.FileMode should not use in windows.
I will remove the if block.